frontend/packages/console-app/console-extensions.json

```diff
@@ -805,6 +805,21 @@
       "prefixNamespaced": true
     }
   },
+  {
+    "type": "console.navigation/resource-ns",
+    "properties": {
+      "id": "operators",
+      "section": "ecosystem",
+      "name": "%console-app~Installed Operators%",
+      "insertAfter":"software catalog",
```

Comment:
`Installed Operators` is in front of `Software Catalog` which seems not expected
<img width="1236" alt="Screenshot 2025-06-24 at 3 01 59 PM" src="https://github.com/user-attachments/assets/d69a2216-3a6a-42b8-bbc1-3b5af3ab82d1" />